src/wings_color.erl

```diff
@@ -227,6 +228,127 @@ col_to_linear(X)
   when is_float(X) ->
     math:pow((X+0.055)/1.055, 2.4).
 
+
+%% hsb_to_rgb
+%% The colors of this function differ a little bit from the hsv one
+%% example: Hue:160 Sat:0.5 Brightness:1.0 -> R:0.5 G:1 B:0.83
+%%
+hsb_to_rgb(H, S, V) when S < 0.0 ->
+    hsb_to_rgb(H, 0, V);
+hsb_to_rgb(H, S, V) when S > 1.0 ->
+    hsb_to_rgb(H, 1.0, V);
+hsb_to_rgb(H, S, V) when V < 0.0 ->
+    hsb_to_rgb(H, S, 0.0);
+hsb_to_rgb(H, S, V) when V > 1.0 ->
+    hsb_to_rgb(H, S, 1.0);
+hsb_to_rgb(H, S, V) when H >= 360.0; H < 0.0 ->
+    hsb_to_rgb(0.0, S, V);
+hsb_to_rgb(H_0, S, V) ->
+    H = H_0 / 60.0,
+    I = floor(H),
+    F = H - I,
+    P = V * (1.0 - S),
+    Q = V * (1.0 - (S * F)),
+    T = V * (1.0 - (S * (1.0 - F))),
+
+    case (I) of
+        0 ->
+            R = V,
+            G = T,
+            B = P;
+        1 ->
+            R = Q,
+            G = V,
+            B = P;
+        2 ->
+            R = P,
+            G = V,
+            B = T;
+        3 ->
+            R = P,
+            G = Q,
+            B = V;
+        4 ->
+            R = T,
+            G = P,
+            B = V;
+        5 ->
+            R = V,
+            G = P,
+            B = Q
+    end,
```

Comment:
Can be made shorter with the following, 
don't know what you prefer we will loose implicit documentation with the R,G,B variables,
but much shorter code, and remove the exporting of variables from the 'case' statement which should be avoided.

```
case I of 
    0 -> {V,T,P};
    1 -> {Q,V,P};
    2 -> {P,V,T};
    3 -> {P,Q,V};
    4 -> {T,P,V};
    5 -> {V,P,Q}
 end.
```